### PR TITLE
MA0134 is correctly reported in local function

### PR DESCRIPTION
**Pull request by jackalvrus**

### CODE REVIEW

**Comment by jackalvrus** [comment]
Fixes #597

File: src/Meziantou.Analyzer/Rules/AwaitAwaitableMethodInSyncMethodAnalyzer.cs [hunk 1]
```diff
@@ -30,9 +30,6 @@ public override void Initialize(AnalysisContext context)
             var awaitableTypes = new AwaitableTypes(context.Compilation);
             context.RegisterSymbolStartAction(context =>
             {
-                if (context.Symbol is IMethodSymbol method && (method.IsAsync || method.IsTopLevelStatementsEntryPointMethod()))
-                    return; // Already handled by CS4014
```
Comment (meziantou) [inline src/Meziantou.Analyzer/Rules/AwaitAwaitableMethodInSyncMethodAnalyzer.cs:34]: Can you move the comment `// Already handled by CS4014` to the line 58?

Also, you can remove the `// While there is a check in RegisterSymbolStartAction, this is needed to handle lambda and delegates` comment as it is not valid anymore.

Comment (jackalvrus) [reply src/Meziantou.Analyzer/Rules/AwaitAwaitableMethodInSyncMethodAnalyzer.cs:34]: Good catch. Comments fixed.

**Comment by meziantou** [comment]
Thanks! The new version (2.0.85) will be available soon :)

**Pull request merged**